tcg/tcg-op-vec.c

```diff
@@ -279,19 +279,19 @@ void vec_gen_4(TCGOpcode opc, TCGType type, unsigned vece,
     op->args[3] = c;
 }
 
-static void vec_gen_6(TCGOpcode opc, TCGType type, unsigned vece, TCGArg r,
-                      TCGArg a, TCGArg b, TCGArg c, TCGArg d, TCGArg e)
-{
-    TCGOp *op = tcg_emit_op(opc, 6);
-    TCGOP_VECL(op) = type - TCG_TYPE_V64;
-    TCGOP_VECE(op) = vece;
-    op->args[0] = r;
-    op->args[1] = a;
-    op->args[2] = b;
-    op->args[3] = c;
-    op->args[4] = d;
-    op->args[5] = e;
-}
+// static void vec_gen_6(TCGOpcode opc, TCGType type, unsigned vece, TCGArg r,
```

Comment:
Can you use C comments like elsewhere ? And add a note on why this is commented out vs removed (I imagine this is unused now but may be useful later ?)

Reply:
It's an effect of -werror, unused symbols will make the compilation fail (it was treated as warning before).
I'll adapt the comment style and add a comment.
It's better to keep it commented imho for future merges with QEMU, maybe this will become used elsewhere one day.